templates/types/streaming/nextjs/app/components/ui/chat/index.ts

```diff
@@ -3,3 +3,30 @@ import ChatMessages from "./chat-messages";
 
 export { type ChatHandler } from "./chat.interface";
 export { ChatInput, ChatMessages };
+
+export enum MessageAnotationType {
+  IMAGE = "image",
+  DOCUMENT = "document",
```

Comment:
how about calling this sources? as we're sending the source nodes?
```suggestion
  SOURCES = "sources",
```